index.html

```diff
@@ -1836,6 +1836,62 @@ <h2>Algorithms</h2>
 measures that could contribute to a safer and more efficient ecosystem.
       </p>
 
+      <section class="normative">
+        <h3>Processing Model</h3>
+
+        <p>
+The processing model used by a [=conforming processor=] and its
+application-specific software is described in this section. When software
+needs to ensure information is tamper-evident, it:
+        </p>
+
+        <ol>
+          <li>
+Arranges the information into a document, such as a JSON or JSON-LD document.
+          </li>
+          <li>
+If the document is a JSON-LD document, selects one or more JSON-LD Contexts
+and expresses them using the `@context` property.
+          </li>
+          <li>
+Selects a cryptography suite that meets the needs of the use case, such as
+one that uses acceptable cryptographic key material and provides full
+disclosure, selective disclosure, or unlinkable disclosure functionality.
```

Comment:
Including [suggestion](https://github.com/w3c/vc-data-integrity/pull/291/files#r1691623442) from @dlongley 
```suggestion
The software selects one or more cryptography suites that meet the needs of
the use case, such as one that provides full, selective, or unlinkable
disclosure, using acceptable cryptographic key material.
```